hack/make-tools.mk

```diff
@@ -7,7 +7,7 @@ endef
 
 CONTROLLER_GEN = $(BIN_DIR)/controller-gen
 controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1)
+	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.0)
```

Comment:
https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.23.0/#gov3-update-controller-tools-from-090-to-092, This needs to be 0.9.2

Reply:
done